marimo/_messaging/streams.py

```diff
@@ -198,14 +206,26 @@ def writelines(self, sequence: Iterable[str]) -> None:  # type: ignore[override]
         for line in sequence:
             self.write(line)
 
+    def start(self) -> None:
+        self._watcher.start()
+        return
+
+    def stop(self) -> None:
+        self._watcher.stop()
+        return
+
+    def pause(self) -> None:
+        self._watcher.pause()
+        return
```

Comment:
These (start, stop, pause) shouldn't be public methods, since this class is exposed to the user (by means of patching `sys.stdout`). Same goes for `Stderr`.